math/Color.cpp

```diff
@@ -265,6 +265,13 @@ void Color4::operator-=(const Color4& in_Color)
 	a -= in_Color.a;
 }
 
+void Color4::operator=(const Color3& in_Color)
+{
+  r = in_Color.r;
+  g = in_Color.g;
+  b = in_Color.b;
```

Comment:
the indentation seems wrong compared to line 264

Reply:
https://github.com/sunglab/StarEngine/pull/3/commits/a83fc43f908aff322ba3d01ccf46d638c91c22f1
modify indentation